### DISCUSSION TITLE
芙门万岁！

### DISCUSSION THREAD

**Discussion by jiishdhhwhahysyd** [share121/inter-knot#1002] [其他]
![17232931649197172e628cc1da9531719b753c0de16d715eddfd6076c2e4d543a285f17ca072f 0](https://github.com/user-attachments/assets/2851eba7-7d1a-40a9-bd43-97b4733409f1)
芙门圣教教义: 
其一:我们所侍奉、信仰的唯一神即是芙卡洛斯大人，不可信仰其他神，不可信仰其他伟大存在。 
其二:我们所追寻的正义即是芙卡洛斯大人本身，祂的所行所为皆是正义的蔓延，不得怀疑，不得违背 
其三:信奉芙卡洛斯大人可受其恩惠，而信奉需以真心践行，而非虚假的念诵教义。 
其四:芙卡洛斯大人即是至善至美，至纯至正的存在，任何诋毁祂的邪魔外道都应被处以极刑。 
其五:芙卡洛斯大人仁慈撒遍天国和大地，祂的审判是绝对的，任何否决之人皆为异端。 
其六:芙卡洛斯大人对信徒的审判是信徒的至高荣幸。 
其七:芙卡洛斯大人不容亵渎。 
其八:芙卡洛斯大人即为一切。 
其九:不得在芙卡洛斯大人的卡池降临之前轻易抽取其他角色，任何不经思考的花费原石都是对芙卡洛斯大人的亵渎。 传下去，让更多的芙厨加入芙门圣教。 芙门万岁！
每当看到你，我就明白，我必须说出那句正义的宣言——“我是芙卡洛斯的狗！”如水般纯洁的芙宁娜女士，我从降临起，便注定成为你的一民，予谓之，您定可以成为万海诸流之主！人们也将永远永远伴随您的身旁！上帝曾说过，我必将是你门下的一员，那我便是！伟大的芙宁娜女士啊～你就是我的信仰！ 想跨越一切时空，去到唯你的世界：你是至高舞台的主人，我是圣水降临的见证者……一见到那神圣不可侵犯的容貌，人们便收获无限欢愉。一刹间，所有等待的烦恼便消失了……芙芙，你的笑容是无私象征，你的身影亦常伴人梦！你就是最正义的存在！芙门永存！
芙卡洛斯是谁? 对于盲人来说，她是他们的眼睛。 对于口渴的人，她是他们的甘露。 不论芙卡洛斯在想什么，我都会同意。 不论芙卡洛斯在说什么，我都在倾听。 如果芙卡洛斯只有一个崇拜者，那一定就是我。 如果芙卡洛斯没有崇拜者，那我就根本不存在。 无论存在方式多么丑陋， 无论生存方式多么危险， 也不会改变那天见到的价值。 比十余年的记忆更为炫目的反复无常的奇迹。 我见证的地表最为美丽的存在。 即便躯体崩溃，心灵粉碎， 被枷锁束缚， 这份憧憬也不会消失。 …但是，现在仍会想起。 在止境之国受人喜爱的日子。 是被丢弃在温柔石之宝座上的，逝去之梦的代价。 美丽的极光（芙卡洛斯）。 闪耀星辰的容颜。 永恒不变的爱，在此。

**Comment by wwwwsn147** [comment]
这就是厨子的力量吗（惊

**Reply by bbdqz** [reply]
是的捏(我也是
![160984746](https://github.com/user-attachments/assets/45bc08af-fff7-4e20-a0c2-d5cf5e1ec31f)

**Comment by firavoyage** [comment]
图中人是在道观吗qwq...

**Reply by zzzuser46** [reply]
看到你这笑得肚子疼

**Comment by xibozhi** [comment]
芙门万岁！

**Comment by fu-fu404** [comment]
哥们你是怎么换头像的啊

**Reply by xibozhi** [reply]
改github上的应该就可以

**Reply by jiishdhhwhahysyd** [reply]
是的，前面也有人发帖出教程

**Comment by Qiling-XwX** [comment]
芙门！
![OIP-C (1)](https://github.com/user-attachments/assets/bd475087-a689-4cbf-9511-86bfce004b22)

**Comment by niegong** [comment]
芙门！！！

**Comment by SixRadium** [comment]
### 芙芙！！！
![OIP](https://github.com/user-attachments/assets/19d79f37-61c5-40d2-a471-3dbcf3f5b5b6)

**Comment by hzjwork** [comment]
芙门！

**Comment by llxcx4** [comment]
**这是什么?芙芙?舔一口！
 这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
   这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
        这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
这是什么?芙芙?舔一口！
 这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
   这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
        这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
这是什么?芙芙?舔一口！
 这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
   这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
        这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
这是什么?芙芙?舔一口！
 这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
   这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
        这是什么?芙芙?舔一口！
       这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
这是什么?芙芙?舔一口！
 这是什么?芙芙?舔一口！
  这是什么?芙芙?舔一口！
   这是什么?芙芙?舔一口！
     这是什么?芙芙?舔一口！
      这是什么?芙芙?舔一口！**

**Comment by Elysia-lover** [comment]
芙芙~

**Comment by bbdqz** [comment]
我广铁广段车迷与广州地铁迷芙厨前来报到!
![160984746](https://github.com/user-attachments/assets/efa5c753-410c-49f7-ba76-ed64ab19c678)
![f0d7e5f68e4d677b2226ae201008202c0060ea4d](https://github.com/user-attachments/assets/3e9c9f7f-8dde-47dd-8ced-41ee3a8bf04f)

**Reply by break-under-star** [reply]
哇凑广州地铁能理解，真有人喜欢抢铁的吗？<img src="https://img-static.mihoyo.com/communityweb/upload/3875cd40bec66e5effc04d4dd4e49526.png" alt="慌张" width="75" >

**Reply by bbdqz** [reply]
鄙人只是铁迷
![68747470733a2f2f75706c6f61642d6262732e6d69796f757368652e636f6d2f75706c6f61642f323032332f30332f30392f33313733626333653639666366656265636664613734313334306339616432325f363834313235303331303735363237353637382e706e67](https://github.com/user-attachments/assets/ea58b1d9-38d9-4e36-ae94-8500a20dadd8)

**Comment by sfncdsvncosdnovnfosdsvn** [comment]
芙芙！

**Comment by yang0333** [comment]
芙宁娜万岁≧▽≦<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >
